### DISCUSSION TITLE
Green Software Students Programme

### DISCUSSION THREAD

**Discussion by adamj89** [Green-Software-Foundation/discussions#17] [Community Working Group]
Proposal arising from the Community Working Group meeting: 

Should we have a GSF Student campaign? Validation through a form. Namrata could set up a landing page. 

How would we reconcile with non-member status?

**Comment by Sealjay** [comment]
Sharing this too:
https://github.com/Green-Software-Foundation/oc/discussions/16

We have a discussion for this in the OC as it also cuts across open source at conferences

**Comment by tmcclell** [comment]
Sarah to organize call with members for thought leadership on this and achievements. Thomas asked what the expected outcomes were. Adams - engage in projects and possible thought leadership. Namrata asked to circle back with Asim about possible outcomes. Thomas volunteered to be a part of he subgroup.
